---
bool success                              # indicate successful run of triggered service
geometry_msgs/TransformStamped tf_wrt_map
string message                            # informational, e.g. for error messages
